netmiko/yotc/yotc.py

```diff
@@ -78,13 +81,16 @@ def check_config_mode(self, check_string=")#", pattern="#"):
     def save_config(self, cmd="wr", confirm=True, confirm_response="y"):
         """Saves Config."""
         self.exit_config_mode()
-        return super().save_config(
-            cmd=cmd, confirm=confirm, confirm_response=confirm_response
+        self.enable()
+        output = self.send_command_timing(cmd, strip_prompt=False, strip_command=False)
+        output += self.send_command_timing(
+            confirm_response, strip_prompt=False, strip_command=False, normalize=False,
         )
+        return output
 
     def cleanup(self):
-        """Don't use 'exit' to disconnect."""
-        pass
+        """Don't use 'exit' to disconnect, and reset command max-lines to default"""
+        self.send_config_set("no command max-lines")
```

Comment:
What is the actual command to gracefully logout from the SSH session?

Reply:
sometimes when device is busy, exit is no use, and busy is often,so just disconnect, "no command max-lines" is to reset the "command max-lines 1000" that typed in ```session_preparation()``` , its permanent